tgstation.dme

```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\map_files\RobinStation\memestation.dmm"

```

Comment:
This isn't how you add new maps. This include breaks the map loader

Reply:
Okay. I've never actually added a map before. How do I do it?